Mathlib/Order/CompleteBooleanAlgebra.lean

```diff
@@ -415,6 +415,21 @@ theorem iSup_inf_of_antitone {ι : Type*} [Preorder ι] [IsDirected ι (swap (·
     (hf : Antitone f) (hg : Antitone g) : ⨆ i, f i ⊓ g i = (⨆ i, f i) ⊓ ⨆ i, g i :=
   @iSup_inf_of_monotone α _ ιᵒᵈ _ _ f g hf.dual_left hg.dual_left
 
+theorem isGreatest_himp {α} [GeneralizedHeytingAlgebra α] (a b : α) :
```

Comment:
Does `#find_,home` suggest any earlier files for this lemma?

Reply:
It suggests all of these: [Mathlib.Data.Int.LeastGreatest, Mathlib.Order.Bounds.Basic, Mathlib.Order.WellFounded, Mathlib.Data.Matrix.CharP, Mathlib.Order.Bounds.Image, Mathlib.Order.Filter.Defs, Mathlib.Algebra.CharP.Pi]
Maybe Order.Bounds.Basic would be a good place?

Reply:
Sounds good; please also move the lemma below this one

Reply:
I'm not really sure where to put these... Should they  go into their own section?